src/fides/api/ops/models/privacy_notice.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import annotations
+
+from collections import defaultdict
+from enum import Enum
+from typing import Any, Dict, Iterable, List, Tuple, Type
+
+from sqlalchemy import Boolean, Column
+from sqlalchemy import Enum as EnumColumn
+from sqlalchemy import Float, ForeignKey, String
+from sqlalchemy.dialects.postgresql import ARRAY
+from sqlalchemy.orm import Session, relationship
+
+from fides.api.ops.common_exceptions import ValidationError
+from fides.lib.db.base_class import Base, FidesBase
+
+
+class PrivacyNoticeRegion(Enum):
+    """
+    Enum is not formalized in the DB because it is subject to frequent change
+    """
+
+    us_ca = "us_ca"  # california
+    us_co = "us_co"  # colorado
+    us_va = "us_va"  # virginia
+    us_ut = "us_ut"  # utah
+    eu_be = "eu_be"  # belgium
```

Comment:
@mfbrown does this look better? :)

Reply:
this looks great to me

Reply:
thanks @pattisdr ! gonna close out and merge for now, @mfbrown if you spot any issues we can address 'em in a follow up :)